### PR TITLE
[IMP] Maintenance: update add new equipment

### PR DESCRIPTION
**Pull request by hojo-odoo**
In version 17.0, two fields have been removed from the Maintenance tab of equipment pages. This PR targets the Add new equipment doc, replacing a picture, and updating the copy to reflect these changes.

### CODE REVIEW

**Comment by hojo-odoo** [comment]
Hey @jero-odoo,

This PR is ready for review.

Thanks,
John

File: content/applications/inventory_and_mrp/maintenance/add_new_equipment.rst [hunk 1]
```diff
@@ -64,24 +64,28 @@ details about the piece of equipment:
...
+frequency of the piece of equipment:
 
-- :guilabel:`Preventive Maintenance Frequency`: specifies how often maintenance should be
-  performed to prevent equipment failure
-- :guilabel:`Maintenance Duration`: the amount of time required to fix the equipment when it fails
-- :guilabel:`Expected Mean Time Between Failure`: the average amount of time that the equipment is
-  expected to operate before failing
+- :guilabel:`Expected Mean Time Between Failure`: the average number of days the equipment is
+  expected to operate between failures. This number can be configured manually.
+- :guilabel:`Mean Time Between Failure`: the average number of days the equipment operates for
```
Comment (jero-odoo) [inline content/applications/inventory_and_mrp/maintenance/add_new_equipment.rst:72]:
```suggestion
- :guilabel:`Mean Time Between Failure`: the average number of days the equipment operates
```

File: content/applications/inventory_and_mrp/maintenance/add_new_equipment.rst [hunk 2]
```diff
@@ -64,24 +64,28 @@ details about the piece of equipment:
...
+  between failures. This number is calculated automatically based on previous failures, and cannot
+  be configured manually.
+- :guilabel:`Estimated Next Failure`: the estimated date on which the equipment experiences its next
```
Comment (jero-odoo) [inline content/applications/inventory_and_mrp/maintenance/add_new_equipment.rst:75]:
```suggestion
- :guilabel:`Estimated Next Failure`: the estimated date the equipment experiences its next
```

File: content/applications/inventory_and_mrp/maintenance/add_new_equipment.rst [hunk 3]
```diff
@@ -64,24 +64,28 @@ details about the piece of equipment:
...
+  failure. This date is calculated automatically based on the data in the :guilabel:`Mean Time
+  Between Failure` and :guilabel:`Latest Failure` fields, and cannot be configured manually.
+- :guilabel:`Latest Failure`: the most recent date on which the equipment failed. This date is based
+  on the creation date of the equipment's most recent maintenance request, and cannot be configured
+  manually.
+- :guilabel:`Mean Time To Repair`: the average number of days it takes to repair the equipment. This
```
Comment (jero-odoo) [inline content/applications/inventory_and_mrp/maintenance/add_new_equipment.rst:81]:
```suggestion
- :guilabel:`Mean Time To Repair`: the average number of days needed to repair the equipment. This
```

File: content/applications/inventory_and_mrp/maintenance/add_new_equipment.rst [hunk 4]
```diff
@@ -64,24 +64,28 @@ details about the piece of equipment:
...
-    :guilabel:`Estimated Next Failure`, :guilabel:`Latest Failure`,
-    and :guilabel:`Mean Time To Repair`. These values are calculated automatically based on
-    maintenance requests if any exist.
+   :alt: The maintenance tab for a piece of equipment.
 
 .. tip::
-    To see the maintenance requests for a piece of equipment, go to the page for the equipment and
-    select :guilabel:`Maintenance` in the top right corner of the form.
+    To see any open maintenance requests for a piece of equipment, go to the page for the equipment
+    and select :guilabel:`Maintenance` smart button at the top of the page.
```
Comment (jero-odoo) [inline content/applications/inventory_and_mrp/maintenance/add_new_equipment.rst:91]:
```suggestion
    and click the :guilabel:`Maintenance` smart button at the top of the page.
```

**Review by jero-odoo — APPROVED** [review]
Hey @hojo-odoo great work, only a few comments (mostly suggestions to make things a bit more concise). Approving for the next round. Thanks!

**Comment by hojo-odoo** [comment]
Hey @ksc-odoo,

This PR is ready for final review.

Thanks,
John

File: content/applications/inventory_and_mrp/maintenance/add_new_equipment.rst [hunk 5]
```diff
@@ -64,24 +64,28 @@ details about the piece of equipment:
...
-  performed to prevent equipment failure
-- :guilabel:`Maintenance Duration`: the amount of time required to fix the equipment when it fails
-- :guilabel:`Expected Mean Time Between Failure`: the average amount of time that the equipment is
-  expected to operate before failing
+- :guilabel:`Expected Mean Time Between Failure`: the average number of days the equipment is
+  expected to operate between failures. This number can be configured manually.
+- :guilabel:`Mean Time Between Failure`: the average number of days the equipment operates between
+  failures. This number is calculated automatically based on previous failures, and cannot
+  be configured manually.
+- :guilabel:`Estimated Next Failure`: the estimated date the equipment experiences its next failure.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/maintenance/add_new_equipment.rst:75]:
```suggestion
- :guilabel:`Estimated Next Failure`: the estimated date the equipment may experience its next failure.
```

File: content/applications/inventory_and_mrp/maintenance/add_new_equipment.rst [hunk 6]
```diff
@@ -64,24 +64,28 @@ details about the piece of equipment:
...
-    :guilabel:`Estimated Next Failure`, :guilabel:`Latest Failure`,
-    and :guilabel:`Mean Time To Repair`. These values are calculated automatically based on
-    maintenance requests if any exist.
+   :alt: The maintenance tab for a piece of equipment.
 
 .. tip::
-    To see the maintenance requests for a piece of equipment, go to the page for the equipment and
-    select :guilabel:`Maintenance` in the top right corner of the form.
+    To see any open maintenance requests for a piece of equipment, go to the page for the equipment
+    and click the :guilabel:`Maintenance` smart button at the top of the page.
```
Comment (ksc-odoo) [inline content/applications/inventory_and_mrp/maintenance/add_new_equipment.rst:91]:
```suggestion
    To see any open maintenance requests for a piece of equipment, go to the page for the equipment,
    and click the :guilabel:`Maintenance` smart button at the top of the page.
```
OPTIONAL: added comma for enhanced sentence pacing

**Review by ksc-odoo — APPROVED** [review]
Hey @hojo-odoo -- just finished my Final Review of this super succinct PR. Nicely done. Approving now. Only have a couple things for you to consider. Once you've given those a look, feel free to tag this for Tech Review. Thanks! 👍

**Comment by hojo-odoo** [comment]
Hey @samueljlieber,

This PR is ready for technical review.

Thanks,
John

**Review by samueljlieber — APPROVED** [review]
Looks great to me @hojo-odoo, nice job on these improvements. 👍
..
@robodoo r+

**Pull request closed without merging**